src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -23,7 +23,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class ControllerResolver implements ControllerResolverInterface
+class ControllerResolver extends ArgumentResolver implements ControllerResolverInterface
```

Comment:
As the min. PHP version is 5.5.9, IMO this could be trait not "normal" class, WDYT?